arch/cpu/gecko/Makefile.commander

```diff
@@ -0,0 +1,16 @@
+COMMANDER ?= commander
```

Comment:
This commit is labelled as docker, I think the Makefiles belong to  some other commit.

Reply:
Done